control/controldisplay/group_counter.go

```diff
@@ -8,23 +8,27 @@ import (
 	"golang.org/x/text/message"
 )
 
+type CounterRendererOptions struct {
+	AddLeadingSpace bool
+}
+
 type CounterRenderer struct {
 	failedControls int
 	totalControls  int
 
 	maxFailedControls int
 	maxTotalControls  int
 
-	addLeadingSpace bool
+	options CounterRendererOptions
 }
 
-func NewCounterRenderer(failedControls, totalControls, maxFailedControls, maxTotalControls int, addLeadingSpace bool) *CounterRenderer {
+func NewCounterRenderer(failedControls, totalControls, maxFailedControls, maxTotalControls int, options CounterRendererOptions) *CounterRenderer {
 	return &CounterRenderer{
 		failedControls:    failedControls,
 		totalControls:     totalControls,
 		maxFailedControls: maxFailedControls,
 		maxTotalControls:  maxTotalControls,
-		addLeadingSpace:   addLeadingSpace,
+		options:           options,
```

Comment:
tbh I was thinking of the options more as a way of passing the data into the function in an extensible way - I think we're fine to keep the `AddLeadingSpace` property on the renderer.

What do you think?